content/en/docs/setup/additional-setup/config-profiles/index.md

```diff
@@ -56,6 +52,13 @@ The components marked as **X** are installed within each profile:
 | &nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;`istio-ingressgateway` | X | X | | | | X |
 | &nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;`istiod` | X | X | X | | | X |
 
+
+### Vendor Profiles
+
+Some additional vendor-specific configuration profiles are also available.
+
+1. **openshift**: used for configuring Istio in an OpenShift environment. 
```

Comment:
@sdake @jwendell The yaml of the openshift profile is quite a bit different than any of the other profiles. Should we do a table of features enabled for it?

Reply:
IMO, not. Those are details that profiles intend to hide.

Reply:
So we hide all of the details of what features a user gets with a profile? Seems like it would be better to use one of the profiles that does document what it's doing instead. I don't think they can stack? Ideally I think we'd have like "install on OpenShift" or "Install on Anthos" with x features.